### DISCUSSION TITLE
Clarifying Installation Instructions

### DISCUSSION THREAD

**Discussion by inkeyes-coding** [tonarino/innernet#237] [Q&A] [answered]
In your installation instructions it says to download the .deb files from the releases page but I've noticed that the last two releases don't actually have any .deb files associated with them. Will those be coming down the road or are we intended to compile the newest versions from source?

Just looking for a little clarity as documentation regarding this project is a little sparse.

**Comment by strohel** [comment, marked as answer]
Hi @inkeyes-coding. Indeed, our README was incorrect. We've fixed that in https://github.com/tonarino/innernet/pull/238. Research on external build systems continues in https://github.com/tonarino/innernet/issues/203.

In the mean time, building from source is the way to go. Please try and feel free to open a new discussion if you run into problems.
